### PR TITLE
Update top-level README and remove docs/README file

### PR DESCRIPTION
**Pull request by ggiguash**
Update top-level README.md file with the proper text and links to the remaining docs.

[USHIFT-73](https://issues.redhat.com//browse/USHIFT-73)

### CODE REVIEW

**Review by mangelajo — APPROVED** [review]
/approve

File: README.md [hunk 1]
```diff
@@ -12,18 +12,16 @@ Edge devices deployed out in the field pose very different operational, environm
 
 We believe these properties should also make MicroShift a great tool for other use cases such as Kubernetes applications development on resource-constrained systems, scale testing, and provisioning of lightweight Kubernetes control planes.
 
-Watch this [end-to-end MicroShift provisioning demo video](https://youtu.be/QOiB8NExtA4) to get a first impression of MicroShift deployed onto a [RHEL for edge computing](https://www.redhat.com/en/technologies/linux-platforms/enterprise-linux/edge-computing) device and managed through [Open Cluster Management](https://github.com/open-cluster-management-io).
-
-**Note: MicroShift is still early days and moving fast. Features are missing. Things break. But you can still help shape it, too.**
-
-## Using MicroShift
```
Comment (fzdarsky) [inline README.md:19]: Would it make sense to keep _some_ pointer back to the "getting started" in the community docs on microshift.io, so those who find MicroShift via GitHub and just want to give it a spin (not building from source) know how to start?

Comment (ggiguash) [reply README.md:19]: Yes, I moved the Getting Started to Community section.

Comment (fzdarsky) [reply README.md:19]: That makes sense, thank you.

**Review by fzdarsky — APPROVED** [review]

**Pull request merged**